--- a/Gemfile
+++ b/Gemfile
@@ -5,7 +5,7 @@ require File.expand_path('../lib/regexp_extensions', FOREMAN_GEMFILE)
 
 source 'https://rubygems.org'
 
-gem 'rails', '3.2.15'
+gem 'rails', '3.2.6'
 gem 'json'
 gem 'rest-client', :require => 'rest_client'
 gem "audited-activerecord", "3.0.0"
@@ -29,7 +29,7 @@ if RUBY_VERSION =~ /^1\.8/
 else
   # Newer version of safemode contains fixes for Ruby 1.9
   gem 'safemode', '~> 1.2'
-  gem 'ruby_parser', '~> 3.0.0'
+  gem 'ruby_parser', '>= 3.0.0'
 end
 
 Dir["#{File.dirname(FOREMAN_GEMFILE)}/bundler.d/*.rb"].each do |bundle|
